apps/src/code-studio/pd/foorm/FoormLibraryEditorManager.jsx

```diff
@@ -106,36 +152,52 @@ class FoormEditorManager extends React.Component {
       });
   }
 
+  initializeNewLibrary = () => {
+    this.props.resetAvailableLibraryQuestions([]);
+    this.props.setLibraryData({
+      name: null,
+      version: null,
+      id: null
+    });
+    this.initializeEmptyCodeMirror();
+  };
+
   initializeEmptyCodeMirror = () => {
     this.props.setLastSaved(null);
     this.props.setSaveError(null);
-    this.updateFormData({
-      questions: {},
-      published: null,
-      formName: null,
-      formVersion: null,
-      formId: null
+
+    this.updateLibraryQuestionData({
+      question: {},
+      libraryQuestionName: null,
+      libraryQuestionId: null
     });
     this.setState({
       showCodeMirror: true,
       hasLoadError: false
     });
   };
 
-  updateFormData = formData => {
-    this.props.setFormData(formData);
+  updateLibraryQuestionData = libraryQuestionData => {
+    this.props.setLibraryQuestionData(libraryQuestionData);
     this.props.setHasError(false);
-    this.props.setLastSavedQuestions(formData['questions']);
-    this.props.resetCodeMirror(formData['questions']);
+    this.props.setLastSavedQuestion(libraryQuestionData['question']);
+    this.props.resetCodeMirror(libraryQuestionData['question']);
+  };
+
+  libraryQuestionOptionsAvailable = () => {
```

Comment:
On LP we also decided to name functions that return booleans with `is` prefix, so this would be `islibraryQuestionOptionsAvailable` which sounds really weird. I'll let you decide if you'd like to change or not :)

Reply:
How about `are`? :)